docs/guides/operator/basic-deployment.adoc

```diff
@@ -344,10 +344,10 @@ You can use those credentials to access the Admin Console or the Admin REST API.
 
 [WARNING]
 ====
-Anyone with the ability to create or edit a Keycloak CR should be a namespace level admin.  
+Anyone with the ability to create or edit Keycloak or KeycloakRealmImport CRs should be a namespace level admin.
```

Comment:
Could you add a similar warning to the KeycloakRealmImport CR guide?

Reply:
Added a similar warning.